rlbot_gui/match_runner/custom_maps.py

```diff
@@ -83,11 +83,18 @@ def convert_custom_map_to_path(custom_map: str) -> Optional[str]:
 
 
 def find_all_custom_maps() -> List[str]:
+    """
+    Ignores maps starting with _
```

Comment:
Seems fine but I'm curious what prompted this

Reply:
The "failed challenge" map isn't really playable so I was using it to hide it from the user.